flipperdevices/flipperzero-firmware
# [FL-1062] Add cli commands for IrDA

# What's new

- Add interrupt processing for CLI command (Ctrl+C)
- Add ir_tx <protocol> <address>  <command> and ir_rx cli commands

# Verification

- Build for f4 and f5 targets
- Verify cli commands

# Checklist (do not modify)
